dev-utils/karma.js

```diff
@@ -130,6 +130,18 @@ function prepareConfig(defaultConfig) {
     defaultConfig.plugins.push('karma-firefox-launcher')
     defaultConfig.browsers.push('Firefox')
   } else if (isJenkins && !isSauce) {
+    defaultConfig.plugins.push('karma-junit-reporter')
+    defaultConfig.junitReporter = {
```

Comment:
shall we merge this PR after #322, Did a small refactoring to keep it simple.

Reply:
Yep, it sounds much better to do it sequentially